outputs.tf

```diff
@@ -29,3 +30,7 @@ output "dynamodb_lock_stream_arn" {
 output "dynamodb_lock_stream_label" {
   value = "${aws_dynamodb_table.tf_backend_state_lock_table.*.stream_label}"
 }
+
+output "s3_kms_key_id" {
+  value = "${var.kms_key_id == "" ? var.kms_key_id : data.aws_kms_key.master.arn}"
```

Comment:
Is the arn different than the id? My understanding of that data source is that it will accept an alias OR an ID, which is a nice convenience provided by terraform (if I understand correctly).

Reply:
i removed this in favor of a simplified kms key specification interface